operator/src/main/java/oracle/kubernetes/operator/steps/ManagedServersUpStep.java

```diff
@@ -273,6 +273,8 @@ Collection<ServerStartupInfo> getStartupInfos() {
       if (startupInfos != null) {
```

Comment:
The behavior of startupInfos seems to have changed, and it would seem simpler to initialize it to an empty list when it is declared, rather than doing so in multiple places. Then you don't need this test.